_config/users.yml

```diff
@@ -113,3 +113,11 @@
   githubUsername: mbiarnes
   employedBy: Red Hat
   ircNickname: mbiarnes
+
+- userId: rbourner
+  fullName: Richard Bourner
+  role: Domain Architect
```

Comment:
Role is in relation to the optaplanner project, so I 'll change it to "Contributor" (or leave it out if you prefer that)